packages/components/src/InputAvatar/InputAvatar.tsx

```diff
@@ -13,7 +13,7 @@ interface InputAvatarProps extends Omit<AvatarProps, "size"> {
    * More info is available at:
    * https://atlantis.getjobber.com/?path=/docs/components-forms-and-inputs-inputfile-docs--page#getuploadparams
    */
-  getUploadParams(file: File): UploadParams | Promise<UploadParams>;
```

Comment:
same thought here, is it possible to maintain the old behavior?

we haven't deprecated the type at all prior to this, so in the even someone was making use of it - it would start to fail going forward

Reply:
I'm ok with keeping the existing behaviour. I will add back UploadParams as one of the valid return types.